salt/modules/boto_ec2.py

```diff
@@ -53,6 +53,7 @@
 
 # Import Salt libs
 import salt.utils.compat
+import salt.utils.exactly_one
```

Comment:
We need to check for stack traces on our test suite....
https://github.com/saltstack/salt/pull/29197